modules/nf-core/survivor/filter/main.nf

```diff
@@ -0,0 +1,46 @@
+process SURVIVOR_FILTER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::survivor=1.0.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/survivor:1.0.7--h9a82719_1':
+        'quay.io/biocontainers/survivor:1.0.7--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(vcf_file) // VCF file to filter
+    path(bed)           // file with regions to ignore (NA to disable)
```

Comment:
Can you merge these two inputs (`tuple val(meta), path(vcf_file), path(bed)`) I think it's worth it here to be able to specify a separate BED file for each input